### ISSUE TITLE
support Frame.get with cffi backend

### ISSUE THREAD

**Issue by minrk** [zeromq/pyzmq#960]
CFFI backend hasn't implemented Frame.get/set, which comes up mainly for the security metadata.

**Comment by subiol** [comment]
I was trying to access the user-id in a auth secured connection. Does this mean that at the moment that information can not be accessed with pyzmq?

**Comment by minrk** [comment]
When using the CFFI backend (i.e. in pypy), yes. The Cython backend does support this metadata, but you must use `frames = socket.recv_multipart(copy=False)` to access it.

**Comment by mattip** [comment]
Is this still actionable?

**Comment by minrk** [comment]
Yes, `Frame.get/set` (should wrap `zmq_msg_get/set`) is not implemented in the CFFI backend. Cython implementation [is here](https://github.com/zeromq/pyzmq/blob/v22.2.1/zmq/backend/cython/message.pyx#L332-L420)
